capstone/src/main/java/com/google/univiz/CollegeScoreCardData/CollegeScoreCardInterface.java

```diff
@@ -0,0 +1,19 @@
+package com.google.univiz.collegescorecarddata;
+
+import java.util.List;
+
+interface CollegeScoreCardInterface {
```

Comment:
In general, I don't think you need to suffix the name of interfaces with `Interface`. Users shouldn't really care if it's an interface or a class.

Reply:
ah ok-updated!